easy_thumbnails/engine.py

```diff
@@ -35,10 +36,14 @@ def process_image(source, processor_options, processors=None):
     return image
 
 
-def save_image(image, destination=None, filename=None, **options):
+def save_image(image, thumbnail_options, destination=None, filename=None):
```

Comment:
It seems a shame to lose `**options` here. There may be other implementations out in the world that use it. Couldn't you add the `keep_icc_profile` as a kwarg?